### ISSUE TITLE
import/export

### ISSUE THREAD

**Issue by matthiasg** [MatAtBread/browserify-nodent#3]
In our project we have decided to already use es6 modules and es7 decorators (to remove duplicate code (dependency injection definitions))

I get an error though about import/export not being allowed.

Error:

```
Error: 'import' and 'export' may appear only with 'sourceType: module' (1:0) (nodent)
import {annotate} from 'angular-annotation-decorator';
^ while parsing file: E:\iris-docs-webapp\webapp\src\domain\rs-document-domain-directive.js
```

**Comment by MatAtBread** [comment]
This is due to default settings in acorn. I'll provide an option to change the default sourceType from 'script' to 'module'. I'll set this as the default for browserify-nodent, unless you want a do a PR to show me how options work in browserify :)

**Comment by matthiasg** [comment]
will look into when i can

**Comment by MatAtBread** [comment]
This should be fixed in the latest version of the browserify plugin (as it was fixed in nodent v2.2.0)
